.gitignore

```diff
@@ -0,0 +1,3 @@
+tests.nix
+shell.nix
```

Comment:
Please remove shell.nix and tests.nix

Reply:
Sorry, you just saw my work in progress. This was never meant to be there. :slightly_smiling_face: